applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -108,6 +110,25 @@ static int headset_conn_index_get(struct bt_conn *conn, uint8_t *index)
 	return -EINVAL;
 }
 
+static void unicast_client_location_cb(struct bt_conn *conn, enum bt_audio_dir dir,
+				       enum bt_audio_location loc)
+{
+	if (loc == BT_AUDIO_LOCATION_SIDE_LEFT) {
+		headset_conn[AUDIO_CHANNEL_LEFT] = conn;
+	} else if (loc == BT_AUDIO_LOCATION_SIDE_RIGHT) {
+		headset_conn[AUDIO_CHANNEL_RIGHT] = conn;
+	} else {
+		LOG_ERR("channel not supported");
```

Comment:
Should we disconnect the connection here, or would it do that by itself if it hasn't got the location set?